src/components/invite/useInvites.js

```diff
@@ -148,14 +149,16 @@ const useCollectBounty = () => {
 }
 
 const useInvited = () => {
+  const gdstore = GDStore.useStore()
   const [invites, setInvites] = useState([])
-  const [level, setLevel] = useState({})
+  const [level, setLevel] = useState(gdstore.get('inviteLevel') ?? {})
   const [totalEarned, setTotalEarned] = useState(0)
 
   const updateData = async () => {
     const user = await goodWallet.invitesContract.methods.users(goodWallet.account).call()
 
     const level = await goodWallet.invitesContract.methods.levels(user.level).call()
+    gdstore.set('inviteLevel')(level)
```

Comment:
1. move everything related to getting / setting 'inviteLevel' onto separate hook
2. use this hook here and in the dashboard component (to preload data)